### ISSUE TITLE
do not enforce multipart

### ISSUE THREAD

**Issue by pebau** [opengeospatial/ogc_api_coverages#20]
[Req 2.3](https://github.com/opengeospatial/ogc_api_coverages/blob/master/OAPI-Coverages/Requirements/requirements_class_path.adoc) says: "A coverage accessed according to requirement 2.2 SHALL be returned as multipart coverage including the DomainSet, RangeType, and RangeSet in the coverage’s Native Format."

Multipart is just one possible encoding, others have their practical relevance (and importance) as well. Also, coverage components enumerated are incomplete. Further, demanding Native Format is contradicting format negotiation.

AFAICS what this req wants to say is: coverage shall be encoded according to the format negotiation - this, however, is already stated elsewhere.

What is open is the (coverage specific) distinction of single-file vs multipart; se WCS 2.0 for its definition. 

Bottom line, rather than reinventing a description hastily we might want to rely on CIS 1.1 which defines it. Hence, suggestion:

"A coverage SHALL be returned encoded as defined in OGC CIS 1.1 and WCS 2.0".

**Comment by Schpidi** [comment]
Teleconference 20190503: Make sure to add examples and explanations in informative part to not make developers need to read CIS and WCS standards.
